test/screenshot/golden.json

```diff
@@ -160,11 +160,11 @@
     }
   },
   "spec/mdc-card/classes/media.html": {
-    "public_url": "https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2018/11/02/14_50_15_747/spec/mdc-card/classes/media.html?utm_source=golden_json",
+    "public_url": "https://storage.googleapis.com/mdc-web-screenshot-tests/kfranqueiro/2019/03/19/21_19_18_527/spec/mdc-card/classes/media.html?utm_source=golden_json",
```

Comment:
The action icon buttons looks more black than it should comparing to spec. We might need to adjust the opacity.

Reply:
This is correct, I mistyped above. The icons are supposed to be medium emphasis. Fixed in latest commits.